fabric-network/test/impl/event/blocklistener.spec.ts

```diff
@@ -508,6 +536,30 @@ describe('block listener', () => {
 			const [actual] = await listener.completePromise;
 			expect(actual.blockNumber).to.equal(event.blockNumber);
 		});
+		it('Timestamp property exist in full block transactionevent', async () => {
+			const event = newFullBlockEventInfo(1);
+			addTransaction(event);
+			listenerOptions = {
+				type: 'full'
+			};
+			await network.addBlockListener(listener, listenerOptions);
+			eventService.sendEvent(event);
+			const actual = await listener.completePromise;
+			expect(actual[0].getTransactionEvents()[0].timestamp).not.null;
```

Comment:
Should check the value is correct, not just that it exists. So if we've defined a `timestamp` Date value for these tests, we could test that the timestamp we get from the event matches our expected `timestamp` value